en/admin-guide/security/index.rst

```diff
@@ -0,0 +1,19 @@
+.. _security:
+
+User Roles
+=======
+
+QGEP comes with a set of predefined roles for a multi user configuration.
+
+The SQL related to these roles creation is stored here : 
+
+https://github.com/QGEP/datamodel/blob/master/12_roles.sql
```

Comment:
I think this is "advanced reading". Most will be happy with the descriptive list. I would therefore put this at the end of the section, maybe even in a `.. note::` block